controllers/lmes/storage.go

```diff
@@ -0,0 +1,90 @@
+package lmes
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	lmesv1alpha1 "github.com/trustyai-explainability/trustyai-service-operator/api/lmes/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+// generateManagedPVCName creates a name for a PVC to be attached to a specific LMEvalJob
+func generateManagedPVCName(job *lmesv1alpha1.LMEvalJob) string {
+	return job.Name + "-pvc"
+}
+
+// generateMagedPVCResource creates a PersistentVolumeClaim for an LMEvalJob
+func generateMagedPVCResource(job *lmesv1alpha1.LMEvalJob) *corev1.PersistentVolumeClaim {
+	pvcName := generateManagedPVCName(job)
+	pvc := corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      pvcName,
+			Namespace: job.Namespace,
```

Comment:
nit: should there be an `OwnerReference` pointing back to the LMEvalJob?

Reply:
Hi @yhwang, `generateMagedPVCResource` would solely generate the managed PVC resource. This PVC definition will always be created with `createPVC` which sets the job as owner with `ctrl.SetControllerReference(job, pvc, r.Scheme)`. Does that fulfill the requirement? I can change it if not.

Reply:
you're right. I missed the code in `createPVC()` function.